Plugin with `include` or `exclude` options fails on Windows

Using either the `include` or `exclude` options causes the plugin to fail on Windows. The plugin uses a string.search call to check if the requested module path matches either of the include/exclude sets, but using string.search implicitly creates an invalid regex on Windows machines due to the module path forward slashes.

Here is an example repo with the issue: https://github.com/DHedgecock/dnwp-example-failure

Running `npx webpack` will result in:

```shell
ERROR in ./src/index.js
Module not found: Error: Can't resolve './Test' in 'C:\Users\username\Documents\GitHub\dnwp-example-failure\src'
 @ ./src/index.js 1:13-30
```

I took a quick run at making updates to the `.search`, including keeping it, but making the string regex safe, but the tests were failing. I'll take a quick look after work later (I don't know if the test setup where escape characters are included is a realistic example?).

--- a/index.js
+++ b/index.js
@@ -18,6 +18,11 @@ module.exports = function (options) {
   };
 };
 
+function stringIncludes(string, maybeString) {
+  // String.includes throws if the argument is not a string
+  return typeof maybeString === 'string' ? string.includes(maybeString) : false;
+}
+
 function doApply(options, resolver) {
   // file type taken from: https://github.com/webpack/enhanced-resolve/blob/v4.0.0/test/plugins.js
   var target = resolver.ensureHook("undescribed-raw-file");
@@ -34,14 +39,14 @@ function doApply(options, resolver) {
   
       // return if path matches with excludes
       if (options.exclude && options.exclude.some(function(exclude) {
-        return dirPath.search(exclude) >= 0;
+        return dirPath.search(exclude) >= 0 || stringIncludes(dirPath, exclude);
       })) {
         return callback();
       }
   
       // return if path doesn't match with includes
       if (options.include && !options.include.some(function(include) {
-        return dirPath.search(include) >= 0;
+        return dirPath.search(include) >= 0 || stringIncludes(dirPath, include);
       })) {
         return callback();
       }

